contrib/hamilton/contrib/user/zilto/nixtla_statsforecast/run.ipynb

```diff
@@ -0,0 +1,485 @@
+       "<polygon fill=\"black\" stroke=\"black\" points=\"2010.53,-177.97 1999.96,-178.61 2008.62,-184.7 2010.53,-177.97\"/>\n",
+       "</g>\n",
+       "<!-- evaluation_metrics -->\n",
+       "<g id=\"node28\" class=\"node\">\n",
+       "<title>evaluation_metrics</title>\n",
+       "<ellipse fill=\"none\" stroke=\"black\" cx=\"2202.89\" cy=\"-162\" rx=\"98.58\" ry=\"18\"/>\n",
+       "<text text-anchor=\"middle\" x=\"2202.89\" y=\"-158.3\" font-family=\"Times,serif\" font-size=\"14.00\">evaluation_metrics</text>\n",
+       "</g>\n",
+       "<!-- evaluation_metrics&#45;&gt;cross_validation_evaluation -->\n",
+       "<g id=\"edge5\" class=\"edge\">\n",
+       "<title>evaluation_metrics&#45;&gt;cross_validation_evaluation</title>\n",
+       "<path fill=\"none\" stroke=\"black\" d=\"M2172.97,-144.76C2155.24,-135.14 2132.55,-122.83 2113.36,-112.42\"/>\n",
+       "<polygon fill=\"black\" stroke=\"black\" points=\"2115,-109.33 2104.54,-107.63 2111.66,-115.48 2115,-109.33\"/>\n",
+       "</g>\n",
+       "</g>\n",
+       "</svg>\n"
+      ],
+      "text/plain": [
+       "<graphviz.graphs.Digraph at 0x7f06284bbfd0>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "dr = (\n",
+    "    driver.Builder()\n",
+    "    .with_modules(nixtla_statsforecast)\n",
+    "    .build()\n",
+    ")\n",
+    "\n",
+    "# create the DAG image\n",
+    "display(dr.display_all_functions(None))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "['base_models',\n",
+      " 'best_model_per_series',\n",
+      " 'cross_validation_evaluation',\n",
+      " 'cross_validation_predictions',\n",
+      " 'dataset',\n",
+      " 'dataset_plot',\n",
+      " 'evaluation_metrics',\n",
+      " 'fallback_model',\n",
+      " 'forecast_harness',\n",
+      " 'inference_plot',\n",
+      " 'inference_predictions',\n",
+      " 'plotting_config',\n",
+      " 'raw_dataset']\n"
+     ]
+    }
+   ],
+   "source": [
+    "final_vars = [v for v in dr.graph.get_nodes() if v._tags.get(\"module\") == \"__init__\"]\n",
+    "\n",
+    "inputs = dict()\n",
+    "\n",
+    "overrides = dict()\n",
+    "\n",
+    "res = dr.execute(\n",
+    "    final_vars=final_vars,\n",
+    "    inputs=inputs,\n",
+    "    overrides=overrides\n",
+    ")\n",
+    "\n",
+    "pprint(list(res.keys()), width=1)"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "venv",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.10.9"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2
+}
```

Comment:
This should probably show something interesting with the output -- this just prints the keys of the available ones...